doc/user/content/sql/explain.md

```diff
@@ -218,7 +218,7 @@ Operator | Meaning | Example
 **CrossJoin** | An alias for a `Join` with an empty predicate (emits all combinations). | `CrossJoin`
 **Reduce** | Groups the input rows by some scalar expressions, reduces each groups using some aggregate functions and produce rows containing the group key and aggregate outputs. | `Reduce group_by=[#0] aggregates=[max((#0 * #1))]`
 **Distinct** | Alias for a `Reduce` with an empty aggregate list. | `Distinct`
-**TopK** | Groups the inputs rows by some scalar expressions, sorts each group using the group key, removes the top `offset` rows in each group and returns the next `limit` rows.| `TopK order_by=[#1 asc nulls_last, #0 desc nulls_first] limit=5 monotonic=false`
+**TopK** | Groups the inputs rows by some scalar expressions, sorts each group using the group key, removes the top `offset` rows in each group and returns the next `limit` rows.| `TopK order_by=[#1 asc nulls_last, #0 desc nulls_first] limit=5`
```

Comment:
```suggestion
**TopK** | Groups the inputs rows by some scalar expressions, sorts each group using the group key, removes the top `offset` rows in each group, and returns the next `limit` rows.| `TopK order_by=[#1 asc nulls_last, #0 desc nulls_first] limit=5`
```

Pre-exists your change, but while you're here, can you please add a final ([oxford](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)) comma?